### DISCUSSION TITLE
Designer/Viewer scaling

### DISCUSSION THREAD

**Discussion by Zego92** [pdfme/pdfme#581] [General]
Hello community
I have a simple question
Just want to clarify the use cases we are using in our project of pdfme library
on the front app, we connect the viewer/designer
We have a way to create and store templates in our db with the next using these templates for generating pdf files.
But sometimes when the generation logic is executed we see a miss positioning of the placeholders, tables etc.
So my next question is
How can we change the scale of the display within the viewer/designer?

**Comment by hand-dot** [comment]
Hey @Zego92 ,
- Tables are currently in beta, so there may be [positioning issues](https://github.com/pdfme/pdfme/issues?q=is%3Aissue+is%3Aopen+label%3Atable).
- At present, it's not possible to programmatically change the display scale within the viewer/designer.
